From 25f131be221c5b2f8cb4f0c2a32f522415bb3bbf Mon Sep 17 00:00:00 2001
From: Christian Marangi <ansuelsmth@gmail.com>
Date: Mon, 17 Oct 2022 20:15:30 +0200
Subject: [PATCH] Mute some no previous prototype compilation warning

Mute some no previous prototype compilation warning found in
atlas_unsafe, rxtxrpt_main and route_set_flags.

Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>
---
 include/libbb.h         | 2 ++
 libbb/route_set_flags.c | 2 ++
 2 files changed, 4 insertions(+)

--- a/include/libbb.h
+++ b/include/libbb.h
@@ -463,6 +463,7 @@ char *is_suffixed_with(const char *strin
 #define ATLAS_TIMESYNC_FILE_REL    ATLAS_DATA_NEW_REL "/timesync.vol"
 #define ATLAS_FUZZING_REL          "data"
 
+extern int atlas_unsafe(void);
 extern char *rebased_validated_filename(const char *path, const char *prefix);
 extern char *rebased_validated_dir(const char *path, const char *prefix);
 extern int validate_atlas_id(const char *atlas_id);
@@ -484,6 +485,7 @@ extern void read_response(int fd, int ty
 extern void read_response_file(FILE *file, int type, size_t *sizep,
 	void *data);
 extern void write_response(FILE *file, int type, size_t size, void *data);
+extern int rxtxrpt_main(int argc, char *argv[]);
 
 int ndelay_on(int fd) FAST_FUNC;
 int ndelay_off(int fd) FAST_FUNC;
--- a/libbb/route_set_flags.c
+++ b/libbb/route_set_flags.c
@@ -1,6 +1,8 @@
 #include <platform.h>
 #include <net/route.h>
 
+#include "libbb.h"
+
 static const
 IF_NOT_FEATURE_IPV6(uint16_t)
 IF_FEATURE_IPV6(unsigned)
